### ISSUE TITLE
Writing my own commands

### ISSUE THREAD

**Issue by Yash117** [huytd/agar.io-clone#358]
I know there was a previous issue that had to do with writing your own commands, but I want to know if there's a guide on the abstractions. For example what's the user object's name value - could be user.name

**Comment by abalabahaha** [comment]
I suggest you take a look at the rest of the code and/or use console.log to determine where properties are and how they should be accessed.

(FYI user.name seems to be correct)

**Comment by Yash117** [comment]
Okay, I was going to do this if there was no guide like a commands.md. Maybe if you're bored or feel like it can do this? :D Thanks for responding.

**Comment by Yash117** [comment]
I mean not a file in the repo obviously, but just some guide somewhere.

**Comment by Yash117** [comment]
^ Is where I wanted to find the original guide... I have to be more specific with my comments lol.

**Comment by abalabahaha** [comment]
Yeah there's no guide at the moment, so you have to rely on existing code...
